runtime/wasmer.go

```diff
@@ -265,6 +266,32 @@ func ext_blake2_256(context unsafe.Pointer, data, length, out int32) {
 //export ext_twox_128
 func ext_twox_128(context unsafe.Pointer, data, len, out int32) {
 	log.Debug("[ext_twox_128] executing...")
+	instanceContext := wasm.IntoInstanceContext(context)
+	memory := instanceContext.Memory().Data()
+
+	// compute xxHash64 twice with seeds 0 and 1 applied on given byte array
+	h0 := xxhash.NewS64(0) // create xxHash with 0 seed
+	_, err := h0.Write(memory[data : data+len])
```

Comment:
Lets add a `log.Debug` somewhere here to log the value being hashed

Reply:
- added `log.Debug` for value and xxH64 results
- Found another implementation to test values: https://www.npmjs.com/package/xxhashjs
    - wrote quick test: 
```var xxhashjs = require("xxhashjs")
var h0 = xxhashjs.h64( 'helloworld', 0x00 )  // seed 0
console.log("res0: " + h0)
var h1 = xxhashjs.h64( 'helloworld', 0x01 )  // seed 1
console.log("res1: " + h1)
```
  - Updated `wasmer_test.go` to check for these values.

Reply:
Update:  changed `wasmer_test.go` to use static know values to test against (instead of generating values dynamically).  Know results are from https://github.com/paritytech/substrate/blob/master/core/executor/src/wasm_executor.rs lines 1554 - 1560.  Tests pass, should be all set now.